package com.stefanini.hackathon.exception;

public class CursoNotFoundException extends Exception{
    public CursoNotFoundException(Long id) {
       super("Curso n√£o encontrado com id:" +id);
    }
}
